<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="obstacle_3">
    <link name="base link">
      <pose>0 0 0 0 0 0</pose>
      <inertial>
        <mass>1.0</mass>
        <inertia>
          <ixx>1e-5</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>1e-5</iyy>
          <iyz>0</iyz>
          <izz>1e-5</izz>
        </inertia>
      </inertial>
      <collision name="collision">
        <geometry>
          <box>
            <size>0.05 0.1 0.12</size>
          </box>
        </geometry>
        <surface>
          <friction>
            <ode>
              <mu>50.0</mu>
              <mu2>50.0</mu2>
            </ode>
          </friction>
          <contact>
          	<ode>
          		<min_depth>0.001</min_depth>
          		<max_vel>0.0</max_vel>
          	</ode>
          </contact>
        </surface>
      </collision>
      <visual name="visual">
        <geometry>
          <box>
            <size>0.05 0.1 0.12</size>
          </box>
        </geometry>
        <material>
          <ambient>0.1 0.1 0.1 1</ambient>
          <diffuse>0.1 0.1 0.1 1</diffuse>
          <specular>0 0 0 0</specular>
          <emissive>0 0 0 1</emissive>
        </material>
      </visual>
    </link>
  </model>
</sdf>
